docusaurus-search-local/src/client/theme/SearchBar/SearchBar.tsx

```diff
@@ -10,7 +10,8 @@ import useDocusaurusContext from "@docusaurus/useDocusaurusContext";
 import ExecutionEnvironment from "@docusaurus/ExecutionEnvironment";
 import { useHistory, useLocation } from "@docusaurus/router";
 import { translate } from "@docusaurus/Translate";
-import { useActiveVersion } from '@docusaurus/plugin-content-docs/client';
+import { useDocsPreferredVersion } from "@docusaurus/theme-common";
```

Comment:
We have moved this export to an internal entrypoint

```js
import {
  useDocsPreferredVersion,
} from '@docusaurus/theme-common/internal';
```

This also means that technically we are not 100% ready to consider this API to be part of our public API surface and make no breaking changes on it 😅 

But this can be discussed if you don't have a better choice than using it, and in practice we may not break it

Reply:
I couldn't find any other way to do it so it is either that or if you know of any other, supported, way of getting the selected version?

Reply:
Will move useDocsPreferredVersion to our public api

Even though we may do a breaking change later, it will be clearly highlighted in a major release blog post